src/components/permissions/rowLevelPolicyHelpers.ts

```diff
@@ -71,7 +71,7 @@ const updateRowPolicies = async () => {
       .map(({ policyname, tablename }: any) => `DROP POLICY "${policyname}" ON "${tablename}"`)
       .join(';'),
   })
-  console.log(newPolicies.join(';'))
+  // console.log(newPolicies.join(';'))
```

Comment:
I think it looks cleaner and easier to follow without this massive bit of text in the console

Reply:
Can this be set as a flag? Like we have on other places to define as displayRowLevelPolicies. Hmmm Although we can easily get access to this using the pg command `select * from pg_policies` so maybe not. 
Maybe leave a comment about that? Instead of this console.log can say `Run select * from pg_policies to check current set of row level permissions created` ?

Reply:
This is for the console log when loading snapshots. It already says:
```
updating row level policies ... 
updating row level policies ... done
```

Isn't that enough for the console? 

> Maybe leave a comment about that? Instead of this console.log can say Run select * from pg_policies to check current set of row level permissions created ?

This doesn't seem relevant to the context of this bit of code, nor to this PR.

Reply:
> Isn't that enough for the console?

Yes that's enough. Could you delete the commented line also please?

All I said was - if the user would like to see what has been included in the database for rowLevel they can run this command. And therefor what this previous command was outputing is not needed.